components/learn/ContentMenuSection.vue

```diff
@@ -3,7 +3,7 @@
     <div class="content-menu-section__container">
       <qiskit-mega-menu-dropdown
         :content.prop="dropdownMenuContent"
-        class="content-menu-section__dropdown bx--col-md-4 bx--col-lg-4 bx--no-gutter"
+        class="content-menu-section__dropdown cds--col-md-4 cds--col-lg-4 bx--no-gutter"
```

Comment:
`cds--no-gutter`

Reply:
Thank you! Changed in https://github.com/Qiskit/qiskit.org/pull/2986/commits/8781eb496a6e8a5bfc9f3fdf4f22fba820102597.